packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/CreateRulesTests.java

```diff
@@ -206,7 +207,7 @@ public void siteContributorCannotCreateRule()
         restClient.assertLastError().containsSummary("Insufficient permissions to manage rules");
     }
 
-    /** Check that a Consumer cannot create a rule in a private folder. */
+    /** Check that a Consumer cannot create a rule in a folder in a private site. */
     public void siteConsumerCannotCreateRule()
```

Comment:
Do we need `@Test (groups = { TestGroup.REST_API, TestGroup.RULES })` annotation for the method to be executed?